solr/core/src/java/org/apache/solr/client/solrj/embedded/EmbeddedSolrServer.java

```diff
@@ -217,62 +218,18 @@ public NamedList<Object> request(SolrRequest<?> request, String coreName)
         throw new SolrException(SolrException.ErrorCode.BAD_REQUEST, "unknown handler: " + path);
       }
       req =
-          _parser.buildRequestFrom(
-              core, params, getContentStreams(request), request.getUserPrincipal());
+          core.getSolrConfig()
+              .getRequestParsers()
```

Comment:
Using a SolrCore's SolrRequestParsers instance instead of a default one on this class means we work with parsers and configuration specific to the core.  This was previously simply overlooked.